packages/docs/docs/guide/getting-started.md

```diff
@@ -1,12 +1,15 @@
 # Getting Started
 
-::: warning Prerequisite
-VuePress requires [Node.js](https://nodejs.org/en/) >= 8.6.
-:::
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/) >= 8.6.
```

Comment:
Node 8.x is not maintained. Maybe we should recommend LTS version in our docs

Reply:
Good point! I'll bump it up to v10 since a lot of people are on it and LTS is not technically a requirement.

```suggestion
- [Node.js 10+](https://nodejs.org/en/)
```